src/Copulas.jl

```diff
@@ -16,6 +16,8 @@ module Copulas
     import LogExpFunctions
     import QuadGK
     import LinearAlgebra
+    using PolyLog
+    using BigCombinatorics
```

Comment:
Please use `import` only and qualify your uses of the functions when you do use them, this improves readability of the code